src/navigation.md

```diff
@@ -63,6 +63,7 @@ search:
 - Dynamic Programming
     - [Introduction to Dynamic Programming](dynamic_programming/intro-to-dp.md)
     - [Knapsack Problem](dynamic_programming/knapsack.md)
+    - [Longest increasing subsequence](sequences/longest_increasing_subsequence.md)
```

Comment:
Please note that the way navigation works currently, each article is assumed to belong to exactly one (sub)section. In particular, this article already appears in "Miscellaneous -> Sequences" section. I don't have a strong opinion on which section it's better to keep it, but if you want to move it here, you should also change the file directory to `dynamic_programming` and create a [redirect](https://cp-algorithms.com/contrib.html#redirects) from the old link.

Reply:
Given the lack of articles in the dp section it probably should be moved as people care more about dp than random sequences (heck, my intro to dp article ranks 6 on duckduckgo right now!). I'll probably look into it.